Task failed with exception: source="task"
TypeError: 'coroutine' object is not iterable
File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py", line 838 in run

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py", line 1130 in _execute_task

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/bases/operator.py", line 408 in wrapper

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow_ai_sdk/operators/agent.py", line 44 in execute

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/bases/operator.py", line 408 in wrapper

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/bases/decorator.py", line 251 in execute

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/bases/operator.py", line 408 in wrapper

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py", line 216 in execute

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py", line 239 in execute_callable

File "/app/fdi/fdi_venv/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py", line 81 in run

File "/app/fdi/dags/dynamic_input_agent.py", line 374 in classify_document
